src/main/java/core/basesyntax/test/Test.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.test;

```

Comment:
It's better to apply below change before proceeding to next task in order not to make confusion with tests that will be added
```suggestion
package core.basesyntax;

```